gitlens/gitlens-release-notes-current.md

```diff
@@ -16,6 +16,63 @@ Features marked with `PREVIEW` require a GitKraken Account, with access level ba
 
 ---
 
+<a id="v15-4"></a>
+
+## Version 15.4
+
+#### Wednesday, September 4, 2024
+
+GitLens 15.4 focuses on branches and worktrees, with icon and tooltip upgrades that make the association between branches and worktrees clearer, improved cleanup when deleting a branch with a worktree or a worktree with a branch, improvements to comparisons, and more.
+
+<img src="/wp-content/uploads/gl-15-4-hero.png" class="img-responsive center img-bordered">
+
+### Branch Icon and Tooltip Improvements
+
+Branches in views and quick pick menus have received a few visual improvements to reduce clutter and make it easier to understand branch state. Branches with worktrees now display a repository icon instead of the typical branch icon, and the tooltip now also indicates that the branch is in a worktree. Branch tooltips now also split out information in a way that is easier to read.
+
+### Worktree Deletion Improvements
+
+When a branch has an associated worktree, it is now easier to clean up the associated branch when deleting the worktree, and vice versa. Using the _Git Delete Worktree_ command on a worktree now also includes options to delete or force-delete both the worktree and its associated branch. Similarly, when deleting a branch that has a worktree, the user is first presented with options for deleting the branch's worktree before deleting the branch itself.
+
+### Branch and Working Tree Comparison Improvements
+
+TBD
```

Comment:
@eamodio - this is essentially about picking a better base branch for comparison, right? If a branch has a PR, it might use the target instead of the branch base?